pkg/identityserver/identityserver_test.go

```diff
@@ -78,12 +78,18 @@ func init() {
 
 	for id, apiKeys := range population.APIKeys {
 		if id.GetUserIDs().GetUserID() == defaultUser.GetUserID() {
+			expiredTime := time.Now().Add(-1 * time.Hour)
 			population.APIKeys[id] = append(
 				apiKeys,
 				&ttnpb.APIKey{
 					Name:   "key without rights",
 					Rights: []ttnpb.Right{ttnpb.RIGHT_SEND_INVITES},
 				},
+				&ttnpb.APIKey{
+					Name:      "expired key",
+					Rights:    []ttnpb.Right{ttnpb.RIGHT_SEND_INVITES},
```

Comment:
```suggestion
					Rights:    []ttnpb.Right{ttnpb.RIGHT_USER_ALL},
```

Nit: to test the expiration in isolation this should at least have sufficient rights to perform the action otherwise.